tests/services/cache-service.test.ts

```diff
@@ -0,0 +1,89 @@
+import {CacheService} from "../../src/services/cache-service";
+
+function sleep(milliseconds) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, milliseconds);
```

Comment:
CacheService takes parameters in seconds though.
(Jupp, I actually did fall right into this trap when writing the test in that other comment.)

... Of course, for most of the test cases we do not need to wait whole seconds, but we probably should use strongly typed time units in such cases.

Reply:
FWIW: I google-d up [this](https://jestjs.io/docs/timer-mocks) but I could not right-away figure out how to use that confidently. Might be a good idea though, to make tests not actually wait.

Reply:
I added support for `TimeComponents` now.